libs/design/src/core/prefix-suffix/prefixable/prefixable-interface.ts

```diff
@@ -0,0 +1,9 @@
+import { DaffPrefixDirective } from '../prefix.directive';
+
+/**
+ * An interface enforcing that a component will manage a given DaffPrefixDirective.
```

Comment:
An interface enforcing that a component has the ability to interact with a given DaffPrefixDirective.

Reply:
Updated in ff3c405

Reply:
Updated in ff3c405